src/ja/common/proxy/command_handler.py

```diff
@@ -17,7 +22,48 @@ class CommandHandler(ABC):
     success property of the Response is True if both validations were passed
     and the Command was executed without error, and is false otherwise.
     """
-    def __init__(self, socket: str):
+    def __init__(self, socket_path: str):
```

Comment:
We should probably add the ServerDatabase as parameter here, otherwise you can't call the `execute` methods ...

Reply:
You can just add an argument for the server database for a sub-class constructor.

Reply:
Ah yes, I forgot this is a generic class.